rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/synonyms/20_synonyms_get.yml

```diff
@@ -55,6 +55,38 @@ setup:
         - synonyms: "bye => goodbye"
           id: "test-id-2"
 
+---
+"Pagination - invalid size and from":
+  - do:
+      catch: bad_request
```

Comment:
Nitpick - do we want to catch the actual error strings that are returned, e.g. `must be a positive integer`?